### PR TITLE
chore(java11): Target Java 11

### PR DESCRIPTION
**Pull request by cfieber**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -3,6 +3,70 @@
   <component name="CompilerConfiguration">
```
Comment (cfieber) [inline .idea/compiler.xml:3]: oops will undo this ...

File: .idea/gradle.xml [hunk 2]
```diff
@@ -1,11 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (cfieber) [inline .idea/gradle.xml:1]: and this..

File: kayenta-core/src/main/java/com/netflix/kayenta/service/MetricSetPairListService.java [hunk 3]
```diff
@@ -35,7 +35,7 @@
   private final StorageServiceRepository storageServiceRepository;
 
   public List<MetricSetPair> loadMetricSetPairList(String accountName, String metricSetPairListId) {
-    String resolvedAccountName = getAccount(accountName);
+    var resolvedAccountName = getAccount(accountName);
```
Comment (cfieber) [inline kayenta-core/src/main/java/com/netflix/kayenta/service/MetricSetPairListService.java:38]: a `var` to ensure we JDK11

File: Dockerfile.compile [hunk 4]
```diff
@@ -1,11 +1,9 @@
 FROM alpine:3.11
 RUN apk add --update \
     openjdk11 \
-    openjdk8 \
     && rm -rf /var/cache/apk
 LABEL maintainer="sig-platform@spinnaker.io"
 ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk
```
Comment (plumpy) [inline Dockerfile.compile:6]: I've been removing this `JAVA_HOME` too... there's only one JDK installed so we don't need it and it feels like one less thing to fix if we change distros or something like that...

**Review by plumpy — APPROVED** [review]
Awesome, thanks!

**Pull request merged**